src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceBeforeCtorInvocation.java

```diff
@@ -0,0 +1,62 @@
+/*
+GenericWhitespace
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.genericwhitespace;
+
+import java.lang.annotation.Target;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.annotation.ElementType.TYPE_USE;
+
+public class InputGenericWhitespaceBeforeCtorInvocation {
+    List<String> music = new ArrayList<>();
+    List todo = new ArrayList<> (); // violation, ''>' is followed by whitespace.'
+
+    Gen<String, Integer> eel = new Gen<String, Integer > (); // 2 violations
```

Comment:
plase reuse this patter to share messages for all violations
https://github.com/checkstyle/checkstyle/commit/e46f933d31fa30385146ef08bb9e2f725d7fc206#diff-a277946d953ad41bfa49f4012b6fd8bd41c6ac618fa61fa23784d7d675908418R38

Reply:
Just to clarify, I need to remove inline violation message "`// 2 violations`", place it in the next line, and replace it with "`// 2 violations above:`" ?

Reply:
Should I do this for all the input files or just for `BeforeCtorInvocation` and `AfterNew` inputs? I think it would be counterproductive for lines with too many violation like this:
https://github.com/checkstyle/checkstyle/blob/0eaeb24626e64991d308fa5739cf8decc18d9b2b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceDefault.java#L23

Reply:
It would be awesome but, we can do it in separate PR, to not grow this update more than required.

Reply:
Done.